pkgs/development/libraries/libgphoto2/default.nix

```diff
@@ -14,7 +15,10 @@
 , libxml2
 , gd
 }:
-
+let
+  inherit (nixos {}) options;
+  group = options.programs.gphoto2.group.value;
```

Comment:
I am pretty sure that this won't work on none nixos systems. We should probably just make this a string.

Reply:
How's the current solution?